packages/react-urql/examples/2-using-subscriptions/src/config.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
```

Comment:
Would adding these to a .env file work out of the box with webpack?


#### .env
```
SERVER_PORT=4000
WS_PORT=4001
```

#### index.tsx
```ts
const subscriptionClient = new SubscriptionClient(
  `ws://localhost:${process.env.WS_PORT}`,
  {}
)
```

Reply:
Webpack doesn't pickup .env files automatically sadly. A package like https://github.com/motdotla/dotenv is needed, and the JS has to start with `require('dotenv').config()`

IMO not worth switching to a .env file for the example because it's another package (`motdotla/dotenv`) for people to learn when their real goal is to learn URQL.

Reply:
I usually don't use a package in order to source env files. Just checked and it looks like CRA and Parcel both support .env files without any additional plugins needed. 

I think this might be another reason to use Parcel/CRA and remove the noise that comes from having to manage a webpack config. If not though, it looks like there are [webpack plugins here](https://webpack.js.org/plugins/environment-plugin/#dotenvplugin).